AltUseVendors/__init__.py

```diff
@@ -106,6 +129,7 @@ def sell_trash(PC: WillowPlayerController, vendor: WillowVendingMachine) -> None
         vendor: The vendor they're selling at.
     """
     PC.GetPawnInventoryManager().SellAllTrash()
+    PC.pawn.PlayAkEvent(unrealsdk.FindObject("AkEvent", AKE_SELL))
```

Comment:
`PC.Pawn` - copy the capitalization when using unreal property names.

Reply:
Done